kura/org.eclipse.kura.rest.packages.provider/OSGI-INF/deployment_rest_service.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Eurotech and/or its affiliates and others
+
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+
+	SPDX-License-Identifier: EPL-2.0
+
+	Contributors:
+	 Eurotech
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.kura.rest.deployment.agent.DeploymentRestService">
+   <implementation class="org.eclipse.kura.internal.rest.deployment.agent.DeploymentRestService"/>
+   <property name="kura.service.pid" type="String" value="org.eclipse.kura.rest.deployment.agent.DeploymentRestService"/>
+   <service>
+      <provide interface="org.eclipse.kura.internal.rest.deployment.agent.DeploymentRestService"/>
+   </service>
+   <reference bind="setDeploymentAgentService" cardinality="1..1" interface="org.eclipse.kura.deployment.agent.DeploymentAgentService" name="DeploymentAgentService" policy="static" unbind="unsetDeploymentAgentService"/>
+   <reference name="DeploymentAdmin"
+              policy="static"
+              cardinality="1..1"
+              bind="setDeploymentAdmin"
+              unbind="unsetDeploymentAdmin"
+              interface="org.osgi.service.deploymentadmin.DeploymentAdmin"/>
+   <reference bind="setUserAdmin" unbind="unsetUserAdmin" cardinality="0..1" interface="org.osgi.service.useradmin.UserAdmin" name="UserAdmin" policy="static"/>
```

Comment:
Could you please format the references in the same way?

Reply:
Fixed in 171aab713b8e7da97fa4264b20bc6d5f0d67c87f